frontend/src/lib/components/Tooltip.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react'
+import { Tooltip as AntdTooltip } from 'antd'
+import { TooltipProps } from 'antd/lib/tooltip'
+import { useDebounce } from 'use-debounce'
+
+const DEFAULT_DELAY = 500 //ms
+
+// CAUTION: any changes here will affect tooltips across the entire app.
+export function Tooltip({ children, visible, ...props }: TooltipProps): JSX.Element {
```

Comment:
I think maybe we should add a prop to allow overriding this and keep the default behavior for specific places where it makes sense.

Reply:
got it I think that would help too. i'll add a `isDefault` prop here